types/aws-lambda/trigger/iot-authorizer.d.ts

```diff
@@ -0,0 +1,48 @@
+import {Handler} from "../handler";
+import {PolicyDocument} from "./api-gateway-authorizer";
```

Comment:
We should probably pull out the certificate/policy stuff to somewhere; they get used all over the place. But not in this PR!